lerna.json

```diff
@@ -10,7 +10,7 @@
       {
         "basePath": "packages/axes/dist",
         "dists": [
-          "demo/dist"
+          "packages/demo/dist"
```

Comment:
It will manage dist by version like cdn.

Check the latest and version folders.